Demo/src/main/java/com/paypal/android/DemoApplication.kt

```diff
@@ -0,0 +1,12 @@
+package com.paypal.android
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class DemoApplication : Application() {
+
+    override fun onCreate() {
```

Comment:
It doesn't look were overriding anything in this `onCreate`. Maybe we can clean it up and add it back when we need?